src/main/resources/reference.conf

```diff
@@ -76,6 +79,18 @@ cassandra-journal {
   # The time to wait before cassandra will remove the thombstones created for deleted entries.
   # cfr. gc_grace_seconds table property documentation on http://www.datastax.com/documentation/cql/3.1/cql/cql_reference/tabProp.html
   gc-grace-seconds = 864000
+  
+  # Configure know tags in this section
```

Comment:
This is indeed a technical limitation, but I didn't find another way. How would a materialized view look like that would support that? You can only add one non-primary-key column. Collections are not supported.